pkg/aspect/listChecker/manager.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package listChecker
+
+import (
+	"fmt"
+
+	listcheckerpb "istio.io/api/istio/config/v1/aspect/listChecker"
+
+	"google.golang.org/genproto/googleapis/rpc/code"
+	"istio.io/mixer/pkg/aspect"
+	"istio.io/mixer/pkg/attribute"
+	"istio.io/mixer/pkg/expr"
+)
+
+const (
+	kind = "istio/listChecker"
+)
+
+type (
+	manager struct{}
+)
+
+// Manager returns "this" aspect Manager
+func Manager() aspect.Manager {
+	return &manager{}
+}
+
+// NewAspect creates a listChecker aspect
+func (m *manager) NewAspect(cfg *aspect.CombinedConfig, ga aspect.Adapter) (aspect.Aspect, error) {
+	aa, ok := ga.(Adapter)
+	if !ok {
+		return nil, fmt.Errorf("Adapter of incorrect type. Expected listChecker.Adapter got %#v %T", ga, ga)
+	}
+	_, ok = cfg.Aspect.TypedParams.(*listcheckerpb.Config)
```

Comment:
This is an internal error.
Mark it as internal; or come up with a new error interface ?